templates/base.html

```diff
@@ -80,11 +81,11 @@
               media="screen"
               href="/static/css/styles.css" />
       </head>
-      <body>
+      <body class="{% block body_class %}{% endblock %} l-site">
```

Comment:
Why are we making this an `l-site'?

Reply:
![image](https://github.com/user-attachments/assets/2f31197f-2a0e-4ee8-ba41-3e61b5048659)
To make the footer sticky

Reply:
I see. Why are we making the footer sticky?

Reply:
Sometimes (like in thank-you page) when the content is less the footer is dangling on screen and it no longer looks a footer.

Reply:
Makes sense :+1: